app/models/course_group.rb

```diff
@@ -0,0 +1,5 @@
+class CourseGroup < ApplicationRecord
+  has_many :courses
+
+  validates :name, presence: true, uniqueness: true
```

Comment:
Could we have some error messages on the validations, even if it's a generic "Enter a unique course group name"? Just to prevent the uninformative Rails defaults ("is taken") popping up when we build the admin UI.

Reply:
Added error message